SteamKit2/SteamKit2/Util/HardwareUtils.cs

```diff
@@ -309,22 +351,22 @@ public void Set3B3( string value )
                 this.KeyValues["3B3"].Value = value;
             }
 
+            
+            // 333 is some sort of user supplied data and is currently unused
```

Comment:
Why duplicate this comment from below? This doesn't seem to add any further value.